Configuration/Eras/python/Era_Phase2C4_timing_cff.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Phase2C4_cff import Phase2C4
+from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
+
+Phase2C4_timing = cms.ModifierChain(Phase2C4, phase2_timing)
```

Comment:
This is for the special workflow where timing info is used in reconstruction. This kind of workflow was used for some TDR production, so I have kept it here in case someone wants to test it. It is available in the upgrade matrix, but not forwarded to the standard matrix (yet).

Reply:
ok, answering to myself, this is built on top of Phase2C4, not the bare Phase2, so hgcalV9 enters through that. For the rest I understand it is a kind of placeholder declaration, with no real use attached to it for the time being